config/faker-generator.js

```diff
@@ -0,0 +1,18 @@
+const _ = require('underscore');
+const fs = require('fs');
+const files = fs.readdirSync('./test/fixtures/config');
+
+_.each(files, (file) => {
+  const entity_factory = require(`../test/fixtures/config/${file}`);
+  const dest = `./test/fixtures/collections/${file.split('.')[0]}.json`;
+
+  const collection = _.times(1000, () => {
+    return entity_factory.generate();
+  });
+
+  fs.writeFile(dest, JSON.stringify(collection), (e) => {
+    if(e) {
+      console.log(e);
```

Comment:
this could be `console.error(e);` or even `throw e;`  `e` is an error string correct?

Reply:
Looks like it's an...error...[object?](https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback)? Those docs say it's throw-able, so I'll just do that